src/main/java/net/openhft/chronicle/bytes/MappedBytes.java

```diff
@@ -66,8 +66,9 @@ public static MappedBytes singleMappedBytes(@NotNull final File file, final long
     }
 
     @NotNull
-    public static MappedBytes singleMappedBytes(@NotNull final File file, final long capacity, boolean readOnly)
-            throws FileNotFoundException, IllegalStateException {
+    public static MappedBytes singleMappedBytes(@NotNull final File file,
+                                                final long capacity,
+                                                final boolean readOnly) throws FileNotFoundException, IllegalStateException {
```

Comment:
Line becomes too long.

Reply:
Yes. Again the problem with IllegalStateException. I have fixed it